mobile/src/navigation/app/common/constants.ts

```diff
@@ -78,6 +81,23 @@ const NAVIGATION_ITEMS: DrawerNavigationList[] = [
       },
     ],
   },
+  {
+    name: HIDDEN_SCREEN_KEY,
```

Comment:
Add isHidden flag insead
```suggestion
    name: APP_SCREEN_NAME.UAM_CONFIGURE_GROUP,
```